Minimal patch to get chromium to compile with widevine support.

Exactly the same as -r1, but we now need to patch
ninja to pretty please not terminate our build.

caveat emptor: it's in no way clear that building chromium this
way is safer, from a security perspective, than whatever Google
Chrome does.

Upstream appears to be cooking up a code-signing trust-chain
which may protect users against malicious cdm blobs; I doubt
we benefit from these using this kludge.  Ideally, someone
would look into this more carefully than I have ... tbh as
soon as I got my "stories" back, I pretty much lost interest :)

-gmt

--
--- a/third_party/widevine/cdm/stub/widevine_cdm_version.h
+++ b/third_party/widevine/cdm/stub/widevine_cdm_version.h
@@ -10,6 +10,7 @@

 #include "third_party/widevine/cdm/widevine_cdm_common.h"

+#define WIDEVINE_CDM_VERSION_STRING "unknown"
 #define WIDEVINE_CDM_AVAILABLE

 #endif  // WIDEVINE_CDM_VERSION_H_
--- a/third_party/widevine/cdm/BUILD.gn
+++ b/third_party/widevine/cdm/BUILD.gn
@@ -11,7 +11,7 @@ import("//third_party/widevine/cdm/widev
 # Internal Cast builds set enable_widevine=true to bring in Widevine support.
 # TODO(xhwang): Support component updated CDM on other platforms and remove this
 # assert.
-assert(!enable_widevine || is_win || is_mac || is_chromecast,
+assert(!enable_widevine || is_win || is_mac || is_chromecast || is_linux,
        "Component updated CDM only supported on Windows and Mac for now.")

 widevine_arch = current_cpu
